lib/features/popupMenu/HorizontalPopupMenu.js

```diff
@@ -0,0 +1,46 @@
+import { insertCSS, waitForElementToExist } from '../../util';
+
+import popupMenuCSS from '../../../assets/popup-menu.css';
+import baseCSS from '../../../assets/base.css';
+
+import {
+  forEach
+} from 'min-dash';
+
+import {
+  queryAll as domQueryAll,
+  query as domQuery
+} from 'min-dom';
+
+export default function HorizontalPopupMenu(eventBus) {
+
+  insertCSS('base', baseCSS);
+  insertCSS('popup-menu', popupMenuCSS);
+
+  eventBus.on('popupMenu.open', () => {
+    waitForElementToExist('.djs-popup').then(this._addGroupSeparator);
+  });
+
+}
+
+HorizontalPopupMenu.prototype._addGroupSeparator = function(e) {
+  const popupMenu = domQuery('.djs-popup');
+
+  if (!popupMenu.classList.contains('color-picker')
+      && !popupMenu.classList.contains('align-elements')) {
+    return;
+  }
+
+  const groups = domQueryAll('.djs-popup-group', popupMenu);
+
+  forEach(groups, function(group) {
+
+    if (group === groups[0]) {
+      return;
+    }
+
+    group.insertAdjacentHTML('beforebegin', '<div class="separator"/>');
+  });
+};
```

Comment:
In order to simplify the code, we could use CSS instead. I wrote this snippet for context-pad, and it could be adjusted to serve popup-menu as well:

```css
.bjs-container .djs-context-pad .group:not(:last-child):after {
    display: block;
    content: "";
    width: 1px;
    height: 18px;
    background: var(--context-pad-divider-color);
    margin: auto 4px;
}
```

Reply:
So there is no need for mutation observer, queries etc.

Reply:
Ah that's interesting indeed. I'll try it out